src/Options/Definitions.js

```diff
@@ -349,6 +342,13 @@ module.exports.ParseServerOptions = {
     env: 'PARSE_SERVER_READ_ONLY_MASTER_KEY',
     help: 'Read-only key, which has the same capabilities as MasterKey without writes',
   },
+  replicaSet: {
```

Comment:
I wonder whether we couldn't determine that from the mongoDB client, once it is connected? For example, what happens if MongoDB is not a replica set, but the option is set to true?

Reply:
I guess the admin replSetGetStatus method should help for that

Reply:
Not everybody has admin permissions we could get it from the database url / options